internal/sidekick/internal/dart/templates/lib/service.mustache

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 {{#Codec.DocLines}}
 {{{.}}}
 {{/Codec.DocLines}}
-class {{Codec.Name}} {
+final class {{Codec.Name}} {
```

Comment:
We'll want to figure out how users should test their code, but agree that we can lock things down more tightly for now, and relax later once we have a clear idea for how users will test.

Reply:
We can talk about it later but I was thinking that we should generate a set of mocks for each service. And you'd use it like:

```dart
final mock = MockGenerativeService(
  generateContent: (request) {
    if (contents == ...) {
      return Future.value(Candidates(candidates: [...]));
    }
    throw StateError('unexpected request: $request');
  }
)
```